components/datepicker/datepicker-inner.ts

```diff
@@ -91,9 +91,8 @@ export class DatePickerInner implements OnInit {
   private onlyCurrentMonth:boolean;
   @Input()
   private shortcutPropagation:boolean;
-  // todo: change type during implementation
   @Input()
-  private customClass:any;
+  private customClass:Array<{date:Date, mode:string, clazz:string}>;
```

Comment:
@edinfazlic all good, but:
- in doc you are mentioning `class:string`
- in code you have `clazz:string`
so what it should be?